blog/en/blog/2023/11/20/release-apache-apisix-3.7.0.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Release Apache APISIX 3.7.0"
+authors:
+  - name: "Xin Rong"
+    title: "Author"
+    url: "https://github.com/AlinsRan"
+    image_url: "https://avatars.githubusercontent.com/u/79972061?v=4"
```

Comment:
BTW, if you want to edit quickly, you can use this link `https://github.com/AlinsRan.png`, and this can auto-update if the photo profile changed in the future^^

Reply:
Ok that's cool, thanks for the review & the tip : )